i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+from typing import Callable
+import torch
+from torch import nn
+
+
+class ConformerPositionwiseFeedForward(nn.Module):
+    """
+    Conformer feedforward module
+    """
+
+    def __init__(self, input_dim: int, hidden_dim: int, dropout: float = 0.1, activation: Callable[[torch.Tensor], torch.Tensor] = nn.SiLU):
```

Comment:
Same comment for default `dropout` value here.
But then, as written in another comment, `dropout` should better not be part of this module.